tests/unit/test-cmd/test.run.js

```diff
@@ -51,6 +61,8 @@ describe('run', () => {
       reloadStrategy: sinon.spy(() => {
         log.debug('fake: reloadStrategy()');
       }),
+      addonReload: sinon.spy(),
+      ExtensionRunner: DefaultExtensionRunner,
```

Comment:
it shouldn't be necessary to pass `ExtensionRunner` explicitly if we are using its default value.